webapp/locales/en.json

```diff
@@ -306,9 +306,9 @@
   "editor": {
     "embed": {
       "always_allow": "Always allow embedded content by third party providers (this setting can be changed any time)",
-      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this video the following provider will likely collect user data:",
+      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to play this content the following provider will likely collect user data:",
       "data_privacy_warning": "Data Privacy Warning!",
-      "play_now": "Watch now"
+      "play_now": "Continue"
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-06T09:09:00Z" title="Thursday, August 6th 2020, 11:09:00 am +02:00">Aug 6, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -306,9 +306,9 @@
   "editor": {
     "embed": {
       "always_allow": "Always allow embedded content by third party providers (this setting can be changed any time)",
-      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this video the following provider will likely collect user data:",
+      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to play this content the following provider will likely collect user data:",
       "data_privacy_warning": "Data Privacy Warning!",
-      "play_now": "Watch now"
+      "play_now": "Continue"
```


Cool you thought about this as well. 😍 @Thiruppathi